### PR TITLE
User import - Part 1

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref #3882

### CODE REVIEW

File: pkg/admin/transport/handler_user_import.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+type UserImportHandler struct {
+	JSON              JSONResponseWriter
+	UserImportService UserImportService
+}
+
+func (h *UserImportHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	var err error
+	var request userimport.Request
+	err = httputil.BindJSONBody(r, w, userimport.RequestSchema.Validator(), &request)
```
Comment (tung2744) [inline pkg/admin/transport/handler_user_import.go:30]: Do we have a limit on the json size?

Comment (louischan-oursky) [reply pkg/admin/transport/handler_user_import.go:30]: Yes. pkg/util/httputil/json.go:16

Comment (louischan-oursky) [reply pkg/admin/transport/handler_user_import.go:30]: But it seems too large for user import. Maybe I refactor a bit to make it configurable.

**Comment by tung2744** [comment]
@louischan-oursky I've tried the api, and found there seems a bug in upsert.
If you set `disabled: false`, and trying to upsert non-disabled user, you will see an error
```
        "errors": [
          {
            "name": "Invalid",
            "reason": "InvalidAccountStatusTransition",
            "message": "invalid account status transition: normal -> normal",
            "code": 400,
            "info": {
              "from": "normal",
              "to": "normal"
            }
          }
        ]
```

File: pkg/admin/transport/handler_user_import.go (hunk 1 above)
Comment (louischan-oursky) [reply pkg/admin/transport/handler_user_import.go]: Fixed.

**Comment by louischan-oursky** [comment]
Fixed the account status bug.

**Pull request merged**